app/controllers/participant_signup_controller.rb

```diff
@@ -0,0 +1,28 @@
+class ParticipantSignupController < ApplicationController
+  before_filter :fetch_participant
+  skip_before_filter :check_authorization	
+
+  def index
+    redirect_to :action => :participant_signup_intake_form
+  end
+
+  def intake_form
+    unless current_user && current_user.person.can_view?(@participant)
```

Comment:
Looking at this as it currently stands, I would refactor this `unless` statement into the `check_authorization` before filter. That way, if we ever add any other actions to this controller, they all have the same authorization logic by default.

So, I would remove the `skip_before_filter` line, and add a `private/protected` `check_authorization` method to this controller.

Also, I would add a check for `@current_user.person_type == "Participant"`